src/Uno.Toolkit.UI/Behaviors/ResourceExtensions.cs

```diff
@@ -36,7 +36,11 @@ private static void OnResourcesChanged(DependencyObject d, DependencyPropertyCha
 		{
 			if (d is Control control && e.NewValue is ResourceDictionary newResources)
 			{
+#if !HAS_UNO
+				control.Resources = newResources.DeepClone();
+#else
 				control.Resources = newResources;
```

Comment:
Is it working properly on Windows with these changes?

Reply:
yes, i tested in a separate project...